locales/fr_FR/pcsx2_Iconized.po

```diff
@@ -668,7 +666,7 @@ msgid ""
 "-1 - Reduces the EE's cyclerate to about 75%.  Mild speedup for most games "
 "with high compatibility."
 msgstr ""
-"-1 - Diminue la fréquence du processeur EE d'environ 33% : amélioration des "
+"-1 - Diminue la fréquence du processeur EE d'environ 33%. Amélioration des "
```

Comment:
Don't you think it's weird that the French translation reduces the cyclerate by 33% while the English original only reduces it by 25%... 😛